### ISSUE TITLE
Augury not showing properties for a component

### ISSUE THREAD

**Issue by walrus95** [rangle/augury#1301]
Augury version (required): 1.16.0
Angular version (required): 4.4.3
Date: 5/16/18
OS: Windows 10

Description of issue:
I load my web application and open it up in Augury. Augury will only occasionally show the properties for a component and their values, and sometimes after having to click around at random a ton. It's simply unreliable, big time. Just now I have been trying to get it to show me something useful for half an hour with no luck. Today is decided to take the day off, apparently, as it did yesterday too. 

Great idea for a tool. When it works, it's helpful. Most days though, it's on vacation, which makes it more trouble than it's worth. If you could get it working consistently, it would be a huge asset for Angular development.

I have found that the Firefox version is much more response. It get bogged down easily in Chrome.

It works a little bit better than the old betarang, but not by much. I never got betarang to work at all.

![myfunstuff](https://user-images.githubusercontent.com/32824014/40128528-d580c18c-58ff-11e8-908b-d7c5759477e0.PNG)

Steps to reproduce:
1.  Use it on an Application of any complexity, and you'll see what I'm talking about.

Additional details:
Get funding from Google to Make Augury Great Again.

**Comment by andrewthauer** [comment]
@walrus95 - @santiago-elustondo published an [Augury Canary build](https://chrome.google.com/webstore/detail/augury-canary-build/leechohfifidanmkioncamdcibhmmcji) that fixed a similar issue that many people have been having. If you wouldn't mind, could you install the canary build and see if this resolves your issue?

**Comment by walrus95** [comment]
Welp, one of my components is showing its state in Augury with Augury Canary, but only one, and it's not the one I need to inspect. I would say Canary gets you about 5% of the way there, so it's an improvement, a small improvement though.

Got a Canary for FireFox?

**Comment by walrus95** [comment]
Update: After a little coaxing, I managed to get Canary to actually show me the state of the component in which I'm interested. Go Canary! Now if we just had a FireFox Canary life would be good. (The Augury for Chrome runs slow, but FF Augury is nice and snappy.)

![bird-2871833_1280](https://user-images.githubusercontent.com/32824014/40133888-f922c19e-590e-11e8-94f9-71224a498b66.jpg)

**Comment by santiago-elustondo** [comment]
What coaxing was necessary?

**Comment by santiago-elustondo** [comment]
Thanks for your input, @walrus95
We will continue to investigate this issue, please let us know if you find any new information, or if you can help us replicate the issue.
We don't currently have a Canary for FF, but we will hopefully release one soon. We only released onto the addon store very recently.

**Comment by walrus95** [comment]
@santiago-elustondo Thank you! I had to click around the component tree a bit to get the properties for the component in which I'm interested to show up, but then I tried it again just now and the proprieties showed up right away without the need to click around first. Unfortunately, it then crashed the Chrome tab right after. I find the FF version much more stable. I have never had Augury crash on FF.
